src/components/ui/icon-button/icon-button.scss

```
@@ -52,12 +52,14 @@
   position: absolute;
   top: 50%;
   left: 50%;
-  display: none;
   margin-top: -1.2em;
   margin-left: 1.6em;
   padding: 0.5em 0.7em;
   font-size: 1.4em;
   white-space: nowrap;
+  display: inline-block;
+  opacity: 0;
+  transition: opacity ease 0.1s;
```

I think you can add the delay to this line and remove it from line #76.

```suggestion
  transition: opacity ease 0.1s 2s;
```

---

Hi, as I understand, we don't need any delay on MouseLeave
The delay is only onMouseOver, and when we leave it disappears instantly. That's the idea :)

---

Yes. And for your previous comment, I think you need to launch the server also using: `make run`

---

Hi, I've just loaded Gitpod and I don't think I can notice any changes from last week. But yeah, let me know and I can have a try from my side when it's time :)